packages/shorebird_cli/lib/src/shorebird_create_app_mixin.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:mason_logger/mason_logger.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+mixin ShorebirdCreateAppMixin on ShorebirdConfigMixin {
+  Future<App> createApp({String? appName}) async {
+    late final String displayName;
+    if (appName == null) {
+      String? defaultAppName;
+      try {
+        defaultAppName = getPubspecYaml()?.name;
+      } catch (_) {}
+
+      displayName = logger.prompt(
+        '${lightGreen.wrap('?')} What is your app name?',
```

Comment:
This is fine, but "How should we refer to this app?" to focus on what we're doing *for* the dev, or "App name:" if you just want to be short.  🤷  https://pink.bikeshed.org/